### PR TITLE
Polio 81 campaign soft delete

### PR DESCRIPTION
**Pull request by SimonDus**

### CODE REVIEW

**Comment by olethanh** [comment]
Don't forget to change the mail sending in commands/weekly_email.py too

**Comment by olethanh** [comment]
nevermind I was looking at the wrong place

File: plugins/polio/management/commands/weekly_email.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def send_notification_email(campaign):
 Timeline tracker Automated message.
     """
 
-    print(email_text)
+    # print(email_text)
```
Comment (madewulf) [inline plugins/polio/management/commands/weekly_email.py:71]: delete this

File: plugins/polio/api.py [hunk 2]
```diff
@@ -186,6 +202,23 @@ def send_notification_email(self, request, pk, **kwargs):
 
         return Response({"message": "email sent"})
 
+    @action(
+        detail=False,
+        methods=["get"],
+        url_path="restorecampaigns/(?P<campaign_id>[a-f0-9]{8}-?[a-f0-9]{4}-?4[a-f0-9]{3}-?[89ab][a-f0-9]{3}-?[a-f0-9]{12})",
```
Comment (madewulf) [inline plugins/polio/api.py:208]: Use a PATCH and no need for url_path

File: plugins/polio/tests/test_base.py [hunk 3]
```diff
@@ -239,6 +243,99 @@ def test_can_only_see_campaigns_within_user_org_units_hierarchy(self):
...
+        self.assertEqual(len(response.json()), 8)
+
+    def test_return_only_active_campaigns(self):
+
+        self.create_multiple_campaigns(25)
+
+        campaigns = Campaign.objects.all()
+
+        for c in campaigns[:13]:
+            c.delete()
```
Comment (madewulf) [inline plugins/polio/tests/test_base.py:309]: Maybe use the api to do at least one delete.

**Pull request merged**